commands/common_args.py

```diff
@@ -0,0 +1,188 @@
+import os
+from typing import List
+
+import click
+from typing_extensions import get_args
+
+from kiliautoml.utils.constants import ModelFrameworkT, ModelNameT
+from kiliautoml.utils.type import AssetStatusT
+
+
+class Options:
+
+    project_id = click.option("--project-id", required=True, help="Kili project ID")
+
+    api_endpoint = click.option(
+        "--api-endpoint",
+        default="https://cloud.kili-technology.com/api/label/v2/graphql",
+        help="Kili Endpoint. Usefull to access staging environment.",
+    )
+
+    api_key = click.option(
+        "--api-key", default=os.environ.get("KILI_API_KEY", ""), help="Kili API Key"
+    )
+
+    model_framework = click.option(
+        "--model-framework",
+        default="pytorch",
+        help="Model framework (eg. pytorch, tensorflow)",
+        type=click.Choice(get_args(ModelFrameworkT)),
+    )
+
+    model_name = click.option(
+        "--model-name",
+        default=None,
+        help="Model name (eg. bert-base-cased)",
+        type=click.Choice(get_args(ModelNameT)),
+    )
+
+    model_repository = click.option(
+        "--model-repository", default=None, help="Model repository (eg. huggingface)"
+    )
+
+    target_job = click.option(
+        "--target-job",
+        default=None,
+        multiple=True,
+        help=(
+            "Add a specific target job on which to train on "
+            "(multiple can be passed if --target-job <job_name> is repeated) "
+            "Example: python train.py --target-job BBOX --target-job CLASSIFICATION"
+        ),
+    )
+
+    max_assets = click.option(
+        "--max-assets",
+        default=None,
+        type=int,
+        help="Maximum number of assets to consider",
+    )
+
+    clear_dataset_cache = click.option(
+        "--clear-dataset-cache",
+        default=False,
+        is_flag=True,
+        help="Tells if the dataset cache must be cleared",
+    )
+
+    batch_size = click.option(
+        "--batch-size",
+        default=8,
+        type=int,
+    )
+
+    verbose = click.option("--verbose", default=0, type=int, help="Verbose level")
+
+    randomize_assets = click.option(
+        "--randomize-assets",
+        default=True,
+        type=bool,
+        help="Wether or not to fetch assets from Kili randomized",
+    )
+
+
+def asset_status_in(default: List[AssetStatusT]):
+    default_string = ",".join(default) if default else None
+    return click.option(
+        "--asset-status-in",
+        default=default_string,  # type: ignore
+        callback=lambda _, __, x: x.upper().split(",") if x else None,
+        help=(
+            "Comma separated (without space) list of Kili asset status to select "
+            "among: 'TODO', 'ONGOING', 'LABELED', 'TO_REVIEW', 'REVIEWED'"
+            "Example: python train.py --asset-status-in TO_REVIEW,REVIEWED "
+        ),
+    )
+
+
+class TrainOptions:
+    epochs = click.option(
+        "--epochs",
+        default=10,
```

Comment:
in general too low to get a low performance? Let's raise to 50, and maybe make it configurable (maybe lower for label_errors)